File: tmp/test/metadata_scala/pipelines/ScExtPip/code/src/main/scala/io/prophecy/pipelines/scextpip/graph/testsg_1/Reformat_2.scala
package io.prophecy.pipelines.scextpip.graph.testsg_1

import io.prophecy.libs._
import io.prophecy.pipelines.scextpip.config.ConfigStore._
import io.prophecy.pipelines.scextpip.udfs.UDFs._
import io.prophecy.pipelines.scextpip.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_2 {
  def apply(spark: SparkSession, in: DataFrame): DataFrame = in
}

File: pipelines/PY_DEL_GIT_PIPELINE1/code/py_del_git_pipeline1/graph/Reformat_2.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from py_del_git_pipeline1.config.ConfigStore import *
from py_del_git_pipeline1.udfs.UDFs import *

def Reformat_2(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0
